### ISSUE TITLE
Add Accessiblity to NavBar Menu Items

### ISSUE THREAD

**Issue by SudharakaP** [opencollective/opencollective#5609]
It seems that currently the NavBar v2 items are not accessible via the `tab` button. Ideally we should be able to navigate to the menu items through tabbing. 

Related to https://github.com/opencollective/opencollective/issues/4356

Reference: https://github.com/opencollective/opencollective-frontend/pull/7780#pullrequestreview-991807235

**Comment by Betree** [comment]
Being our main navbar, I think the lack of accessibility is an important problem - actually a blocker for the release IMO. It's not just about the tab button, the inability to focus elements will prevent this from working with screen readers or other accessibility software.

**Comment by SudharakaP** [comment]
> Being our main navbar, I think the lack of accessibility is an important problem - actually a blocker for the release IMO. It's not just about the tab button, the inability to focus elements will prevent this from working with screen readers or other accessibility software.

Working on this. I realize it's not just for these new menu items though. Even out profile dropdown doesn't have this same problem. We cannot open nor select items within it using the `tab`. 🤔 

![image](https://user-images.githubusercontent.com/12435965/171634831-8d09d8fa-1ab0-449c-a68a-3c702d576631.png)

**Comment by SudharakaP** [comment]
@Betree : I've opened a PR for the nav bar menu item accessibility issue. Note that this provides basic accessibility support and the user have to click `return` on the button to open the menus. I think this is good for now? Ideally it would be nice to support opening and closing the menus on `focus` and `blur` events I think, but that seemed somehow a bit complicated. 🤔
